x-pack/plugins/fleet/public/applications/integrations/hooks/use_package_install.tsx

```diff
@@ -66,12 +67,22 @@ function usePackageInstall({
   );
 
   const installPackage = useCallback(
```

Comment:
I am wondering if that PR could be a good candidate to start a unit test for that hook?

Reply:
I attempted to start some unit tests for this hook, but the composition is unique in our codebase (uses `createContainer`) and I wasn't able to quickly add tests. I opened https://github.com/elastic/kibana/issues/136155 as a tech debt item to unblock this from going in.